backend/coins/btc/account.go

```diff
@@ -299,6 +299,7 @@ func (account *Account) Info() *accounts.Info {
 			account.signingConfiguration.ScriptType(),
 			account.signingConfiguration.AbsoluteKeypath(),
 			xpubs,
+			"",
```

Comment:
should be `account.signingConfiguration.Address()` (it is doing a copy of the signing configuration)